dialog/file_test.go

```diff
@@ -129,6 +129,8 @@ func TestShowFileOpen(t *testing.T) {
 	nameLabel := ui.Objects[2].(*fyne.Container).Objects[1].(*widget.ScrollContainer).Content.(*widget.Label)
 	buttons := ui.Objects[2].(*fyne.Container).Objects[0].(*widget.Box)
 	open := buttons.Children[1].(*widget.Button)
+	hidden := ui.Objects[2].(*fyne.Container).Objects[2].(*widget.Check)
+	assert.Equal(t, hidden.Text, "Show Hidden Files")
```

Comment:
This isn't really testing the functionality, just the presentation.
It would be preferrable to have a hidden file in the testdata directory and show that it isn't visible by default, then tapping the checkbox makes it appear.